Formula/irssi.rb

```diff
@@ -49,39 +37,21 @@ def install
       --with-proxy
       --enable-true-color
       --with-socks=no
-      --with-perl=yes
-      --with-perl-lib=#{lib}/perl5/site_perl
+      --with-perl=no
```

Comment:
Is building the Perl module broken?

Reply:
yeah, it is broken.

Reply:
once the meson build issue is fixed, we can try meson build to see it helps. (the master build is already with meson)